front/app/components/PostShowComponents/SharingModalContent/index.tsx

```diff
@@ -242,43 +168,84 @@ class SharingModalContent extends PureComponent<
       whatsAppMessage
     ) {
       return (
-        <Container className={className}>
-          <Rocket src={rocket} alt="" />
-          <Title className={`e2e-${postType}-social-sharing-modal-title`}>
+        <Box
+          width="100%"
+          display="flex"
+          flexDirection="column"
+          alignItems="center"
+          justifyContent="center"
+          className={className}
+        >
+          <img
+            width="40px"
+            height="40px"
+            src={rocket}
+            alt={messages.rocketAltValue.toString()}
+          />
+          <Title
+            fontSize="xxxxl"
+            textAlign="center"
+            m="0"
+            mt="20px"
+            mb="12px"
+            p="0"
+            style={{ lineHeight: '40px' }}
+            className={`e2e-${postType}-social-sharing-modal-title`}
+          >
             {title}
           </Title>
-          <Description>{subtitle}</Description>
-          <SharingWrapper>
-            <SharingButtons
-              context={postType}
-              isInModal
-              url={postUrl}
-              twitterMessage={formatMessage(messages.twitterMessage, {
-                postTitle,
-              })}
-              whatsAppMessage={formatMessage(whatsAppMessage, {
-                postTitle,
-              })}
-              emailSubject={formatMessage(emailSharingSubject, { postTitle })}
-              emailBody={formatMessage(emailSharingBody, {
-                postTitle,
-                postUrl,
-              })}
-              utmParams={{
-                source: `share_${postType}`,
-                campaign: `${postType}flow`,
-                content: authUser.id,
-              }}
-            />
-          </SharingWrapper>
-        </Container>
+          <Text
+            width="100%"
+            maxWidth="500px"
+            color="text"
+            margin="0"
+            mt="12px"
+            mb="52px"
+            p="0"
+            fontSize={'m'}
+            textAlign="center"
+            style={{ lineHeight: '25px' }}
+          >
```

Comment:
There were a lot of extra styles on the components (e.g. line height). Should I check with Ben about removing them?